modules/kubernetes-addons/crossplane/locals.tf

```diff
@@ -0,0 +1,39 @@
+locals {
+  name                 = "crossplane-system"
```

Comment:
should we call this var `namespace`? Since it is used for namespace values? Maybe need a `name` and a `namespace` var?

Reply:
You are right. Changed it now